nanopub/setup_profile.py

```diff
@@ -121,21 +120,24 @@ def _delete_keys():
     os.remove(DEFAULT_PRIVATE_KEY_PATH)
 
 
-def _create_this_is_me_rdf(orcid_id: str, public_key: str, name: str) -> Tuple[Graph, BNode]:
+def _create_this_is_me_rdf(orcid_id: str, public_key: str, name: str
+                           ) -> Tuple[rdflib.Graph, rdflib.BNode]:
     """
     Create a set of RDF triples declaring the existence of the user with associated ORCID iD.
     """
-    my_assertion = Graph()
+    assertion = rdflib.Graph()
+    assertion.bind('foaf', rdflib.FOAF)
+    assertion.bind("npx", namespaces.NPX)
 
-    key_declaration = BNode('keyDeclaration')
-    orcid_node = URIRef(orcid_id)
+    key_declaration = rdflib.BNode('keyDeclaration')
```

Comment:
This reminds me that I really need to document how ```nanopub``` interprets blank nodes somewhere, since it's quite an important feature.

Reply:
I've opened an issue for it (#53)

Reply:
Check